File: api/csharp/CloudAPI/Models/GameDevice.cs
namespace CloudApi.Models
{
    /// <summary>
    ///   An internet button device that may participate in a game of
    ///   Button Pong.
    /// </summary>
    /// 
    public class GameDevice
    {
        /// <summary>The unique identifier of the device.</summary>
        public string DeviceId;

        /// <summary>The access token to be used for communicating with the device for game operations.</summary>
        public string AccessToken;
    }
}

File: api/node/shared/models/gameDevice.js
/**
 * An internet button device that may participate in a game of
 * Button Pong.
 * @class
 */
class GameDevice {

    /**
     * Creates an instance of the class, optionally overriding the default
     * attribute values.
     *
     * @param { object } attributes  The set of values for populating the instance attributes.
     */
    constructor(attributes = null) {
        this.deviceId    = null;
        this.accessToken = null;

        if (attributes) {
            Object.assign(this, attributes);
        }
    }
}

module.exports = GameDevice;